docs/services-web-app.md

```diff
@@ -12,3 +32,192 @@ access to private repos and increases API quotas. To set up a personal access to
 1. Select all repo scopes
 1. Copy the `/services/web-app/.env.example` file and rename to `/services/web-app/.env.local`
 1. Add your token to that new `.env.local` file
+
+### IBMid Authentication Variables
+
+**_Prerequisite:_** `.env.local` file for local development (see
+[Setting up local environment variables](#setting-up-local-environment-variables))
+
+Set the following variable values in `.env.local` for correct IBMid authentication flow:
+
+- IBM_VERIFY_CLIENT_ID=[client id tied to App registration on SSO provisioner (get this from dev
+  team)]
+- IBM_VERIFY_CLIENT_SECRET=[client secret tied to App registration on SSO provisioner (get this from
+  dev team)]
```

Comment:
I think it'd be better to keep these docs all in the auth service and then just point from here to that doc regarding the setup you need to do from an auth perspective.

Something like:
> Setting up local environment variables:
> ### Auth:
> This service depends on the auth package. See (insert some link here) for details on how to set up the auth package.
> ### Some other dep:
> This services depends on abcdefg. See (...) for details on how to set that up

Reply:
Done!